workflow/rocoto_funcs/prep_ic.py

```diff
@@ -22,10 +22,12 @@ def prep_ic(xmlFile, expdir, do_ensemble=False, spinup_mode=0):
         cycledefs = 'prod'
     coldhrs = os.getenv('COLDSTART_CYCS', '03 15')
     cyc_interval = os.getenv('CYC_INTERVAL')
+    sfc_cycling_hrs = os.getenv('SFC_CYCLING_HRS', '99')
 
     # Task-specific EnVars beyond the task_common_vars
     dcTaskEnv = {
-        'COLDSTART_CYCS': f'{coldhrs}'
+        'COLDSTART_CYCS': f'{coldhrs}',
+        'SFC_CYCLING_HRS': f'{sfc_cycling_hrs}',
```

Comment:
It may be more consistent with the existing `COLDSTART_CYCS` if we use something like `SFC_UPDATE_CYCS`?

With that said, I am fine with `SFC_CYCLING_HRS`.